lmnet/lmnet/blocks.py

```diff
@@ -40,7 +40,7 @@ def activation(x): return tf.nn.leaky_relu(x, alpha=0.1, name="leaky_relu")
 
         conv = conv2d("conv", inputs, filters=filters, kernel_size=kernel_size,
                       activation=None, use_bias=False, data_format=channel_data_format,
-                      kernel_initializer=tf.contrib.layers.variance_scaling_initializer(),)  # he initializer
+                      kernel_initializer=tf.variance_scaling_initializer,)  # he initializer
```

Comment:
According to the reference manual, `tf.contrib.layers.variance_scaling_initializer` is he initializer (MSRA initializer), but `tf.variance_scaling_initializer` is not. (Latter doesn't have an explicit short name, Caffe's default.)

- https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/contrib/layers/variance_scaling_initializer
- https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/initializers/variance_scaling

I think it's better to keep `tf.contrib.layers.variance_scaling_initializer()`, since he initializer is major than `tf.variance_scaling_initializer`'s default setting.

Reply:
I see. In that case we can keep it for now. But it will be deprecated in TF2.0, until than we can use it.
 
![Screenshot from 2020-01-29 16-35-16](https://user-images.githubusercontent.com/3903030/73336665-c03da280-42b5-11ea-845d-ac5d7dca3b47.png)